src/components/Categories/Categories.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react'
+import {  createStyles, makeStyles, Theme,Typography,Divider,
+         Box,Button,TableContainer,Table,Tabs,TableRow} from '@material-ui/core';
+import AddCircleIcon from '@material-ui/icons/AddCircle';
+
+const useStyles = makeStyles((theme: Theme) => createStyles({
+    root: {
+        marginLeft:"100px",
+        marginTop:"90px",
+        height:"310px",
+        width:"270px",
+        border:"1px solid #7878a1",
+        borderRadius:"5px",
+    },
+    table:{
+        maxHeight:253,
+    },
+    tab:{
+        width:'100%',
+        height:'35px',
+        justifyContent: 'right',
+        textTransform:'none',
+        borderRadius: '8px',
+        margin: '0 0 10px ',
+    }, 
+}));
+
+const Categories: React.FC = () => {
+
+    function createData(name: string) {
+    return { name };
+    }
```

Comment:
Don't create functions with "function". Linter will show warnings/errors about it since tomorrow.
Use:
`const createData = (name: string) => { return { name } }`